lib/Db/Alias.php

```diff
@@ -49,4 +51,13 @@ public function __construct() {
 		$this->addType('name', 'string');
 		$this->addType('alias', 'string');
 	}
+
+	public function jsonSerialize() {
```

Comment:
you mad add a `: array` return type hint here

Reply:
Done